src/sections/ObjectsList/ObjectsList.js

```diff
@@ -103,14 +112,32 @@ const ObjectsList = () => {
             loadObjects(false);
         }
 
+        console.log(queryOptions);
+
     }, [queryOptions]);
 
+    React.useEffect(() => {
+
+        if(objects !== null){
```

Comment:
بقیه‌ی جاها استایل کد نویسی اینجوری هست، دلیلی داره که اینجا متفاوت نوشته شده؟
```suggestion
        if(objects){
```

Reply:
خوانایی کد بالا میره. می خواستم اطمینان حاصل بشه که مخالف نال هست.